graphprot/foutnet.py

```diff
@@ -90,8 +90,8 @@ def forward(self, x, edge_index):
             index = edge_index[:, edge_index[0, :] == n][1, :]
             gamma[n, :] = torch.mean(beta[index, :], dim=0)
 
-        # alpha = alpha + beta
-        alpha = alpha + beta
+        # alpha = alpha + gamma
```

Comment:
I know I wrote it but this might be the definition of a useless comment :) 
we should remove it later !